packages/stark-core/src/modules/http/services/http.service.spec.ts

```diff
@@ -149,9 +150,11 @@ describe("Service: StarkHttpService", () => {
 	};
 
 	interface HttpRequestOptions {
-		params: {
-			[param: string]: string | string[];
-		};
+		params:
+			| {
+					[param: string]: string | string[];
+			  }
+			| HttpParams;
```

Comment:
Shouldn't the type of this property be simply `params: HttpParams` ?

Reply:
The complete angular options are:
```
options?: {
    headers?: HttpHeaders | {[header: string]: string | string[]},
    observe?: 'body',
    params?: HttpParams|{[param: string]: string | string[]},
    reportProgress?: boolean,
    responseType?: 'json',
    withCredentials?: boolean,
  }
```

But it seems less generic and more specific to our implementation. So I think you are right. Maybe I should also rename it to something like `StarkHttpRequestOptions`? 🤔

Reply:
Now that I look at it in more detail, the way you have it now is correct... because it matches the typings from Angular HttpClient... so never mind... don't care about this comment ;)